soc/nxp/imxrt/Kconfig

```diff
@@ -114,6 +114,24 @@ config DEVICE_CONFIGURATION_DATA
 	  initialized at boot time.
 
 
+config EXTERNAL_MEM_CONFIG_DATA
+	bool "External Memory Configuration Data"
+	help
+	  External memory configuration data (XMDC) provides an alternative
+	  configuration sequences which allows to intilialize the external memory
+	  at the boot time. This sequence allows to configure
+	  external memories (such as SDRAM) with more advanced option.
+	  This is a new alternative boot header compared to DCD, and DCD must be disabled
+	  in order to select this option.
```

Comment:
If DCD must be disabled when this option is enabled then shouldn't the DEVICE_CONFIG_DATA have a "depends on !EXTERNAL_MEM_CONFIG_DATA"?

Reply:
Hi David, 
We cannot set DEVICE_CONFIG_DATA, "depends on !EXTERNAL_MEM_CONFIG_DATA" as this KConfig is shared between different platform like RT10xx.

Reply:
You could set `EXTERNAL_MEM_CONFIG_DATA` to depend on `!DEVICE_CONFIGURATION_DATA` though, right?

Reply:
Yes sure, i've modified as your suggestion. Thanks